appsync-cognito-lambda-cdk/README.md

```diff
@@ -12,80 +13,68 @@ Important: this application uses various AWS services and there are costs associ
 - [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
 - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
 - [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-- [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+- [AWS Cloud Development Kit](https://docs.aws.amazon.com/cdk/api/v2/) (AWS CDK) installed
 
 ## Deployment Instructions
 
-1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+1. Clone the ServerlessLand repository
    ```
    git clone https://github.com/aws-samples/serverless-patterns
    ```
 2. Change directory to the pattern directory:
+
    ```
    cd ./serverless-patterns/cognito-appsync-lambda-cdk/
    ```
-3. Navigate to `/cognito-appsync-lambda-cdk/bin/cognito-auth-cdk.ts` and change the `account` and `region` environment variables to suit your values.
-
-```typescript
-env: { account: "13xxxxxxxxxx", region: "us-east-2" },
-```
 
-5. From the command line, use AWS CDK to deploy the AWS resources for the pattern as specified in the `./cognito-appsync-lambda-cdk/lib/cognito-auth-cdk-stack.ts` file:
+3. From the command line, use AWS CDK to deploy the AWS resources for the pattern as specified in the `./cognito-appsync-lambda-cdk/lib/cognito-auth-cdk-stack.ts` file:
 
    ```bash
-   cdk deploy
+   npx aws-cdk deploy
```

Comment:
before deploying it is necesary to do `npm i `
please add this to the instructions